index.html

```diff
@@ -3981,6 +4080,111 @@ <h3>
             </li>
           </ol>
         </section>
+        <section>
+          <h2>
+            <dfn data-lt="requestShippingAddress(addressParts)">requestShippingAddress()</dfn> method
+          </h2>
+          <aside class="note">
+            <p>
+              Detailed shipping address contains sensitive user information.
+              Developers are encouraged to use {{requestShippingAddress()}}
+              to incrementally request pieces they require instead of asking for
+              all up front using the {{PaymentOptions.requestShipping}}
+              boolean. For example, if flat-rate shipping is available in
+              country A but only variable per-region shipping is available in
+              country B, a developer should request only "country" first and
+              decide if "region" is needed.
+            </p>
+            <p>
+              A future version of this specification may remove the all-in-one
+              {{PaymentOptions.requestShipping}} boolean.
+            </p>
```

Comment:
@danyao, I think we should delete this paragraph. It is too unknown at this point.